src/Rings/mpoly-ideals.jl

```diff
@@ -1325,3 +1326,61 @@ function minimal_generating_set(I::MPolyIdeal{<:MPolyElem_dec}; ordering::Monomi
 
   return elem_type(R)[ R(f) for f in gensS ]
 end
+
+################################################################################
+#
+# Grassman Pluecker Ideal
+#
+################################################################################
+@doc Markdown.doc"""
+    grassman_pluecker_ideal(subspace_dimension::Int, ambient_dimension::Int)
+
+Given an ambient dimension and a subspace dimenson dimension return the ideal
+generated by the plucker relations in the Polynomial Ring over the Rationals. 
+
+# Examples
+```jldoctest
+julia> grassman_pluecker_ideal(2, 4)
+ideal(x[0]*x[5] - x[1]*x[4] + x[2]*x[3])
+```
+"""
+function grassman_pluecker_ideal(subspace_dimension::Int, ambient_dimension::Int)
+    return convert(Polymake.ideal.pluecker_ideal(subspace_dimension, ambient_dimension))
+end
```

Comment:
Isn't this method redundant? The next method also coverers the two argument case, doesn't it?